clientlib/src/main/proto/yelp/nrtsearch/luceneserver.proto

```diff
@@ -458,6 +458,8 @@ message LiveSettingsRequest {
     int32 defaultTerminateAfter = 14;
     //Terminate after max recall count value to use when not specified in the search request.
     int32 defaultTerminateAfterMaxRecallCount = 15;
+    // Controls the maximum percentage of deleted documents that is tolerated in the index
+    int32 deletePctAllowed = 16;
```

Comment:
This value is represented by an integer here, but it appears to be a double in the lucene api